ScoutSuite/providers/aws/services/rds.py

```diff
@@ -138,11 +136,10 @@ def get_security_groups_info(rds_client, region_info):
     for group in groups:
         region_info['vpcs'][ec2_classic]['security_groups'][group['DBSecurityGroupName']] = parse_security_group(group)
 
+
 def parse_security_group(group):
-    security_group = {}
-    security_group['name'] = group['DBSecurityGroupName']
-    security_group['description'] = group['DBSecurityGroupDescription']
-    security_group['ec2_groups'] = {}
+    security_group = {'name': group['DBSecurityGroupName'], 'description': group['DBSecurityGroupDescription'],
```

Comment:
Once more here, we should keep it consistent across all services (aka. how it was before).

Reply:
This way is much cleaner and PyCharm suggests it as a correction though...

Reply:
Same asnwer as above